### PR TITLE
Translate to Spanish

### PR DESCRIPTION
**Pull request by rotvulpix**

### CODE REVIEW

File: README-es.md [hunk 1]
```diff
@@ -0,0 +1,638 @@
...
+## Tabla de Contenidos
+
+1. [Query Selector](#query-selector)
+1. [CSS & Estilo](#css--style)
+1. [Manipulación DOM](#dom-manipulation)
+1. [Ajax](#ajax)
+1. [Eventos](#events)
+1. [Utilidades](#utilities)
+1. [Traducción](#translation)
+1. [Soporte de Navegadores](#browser-support)
```
Comment (camsong) [inline README-es.md]: these anchors link are broken

File: README-es.md [hunk 2]
```diff
@@ -0,0 +1,638 @@
...
+
+  ```js
+  // jQuery
+  $(window).scrollTop();
+
+  // Nativo
+  (document.documentElement && document.documentElement.scrollTop) || document.body.scrollTop;
+  ```
+
+**[⬆ back to top](#table-of-contents)**
```
Comment (camsong) [inline README-es.md]: link broken

File: README-es.md (hunk 1 above)
Comment (rotvulpix) [inline README-es.md]: let me see it.

File: README-es.md [hunk 3]
```diff
@@ -0,0 +1,638 @@
...
+  ```js
+  // jQuery
+  $.contains(el, child);
+
+  // Nativo
+  el !== child && el.contains(child);
+  ```
+
+**[⬆ back to top](#table-of-contents)**
+
```
Comment (camsong) [inline README-es.md]: `Alternatives` part here

Comment (rotvulpix) [inline README-es.md]: please check :)

**Comment by camsong** [comment]
I don't know Spanish but I will believe you :)

**Comment by rotvulpix** [comment]
I'm from Chile :heart:

**Pull request merged**